cypher/cypher-docs/src/docs/graphgists/query-tuning/advanced-query-tuning-example.asciidoc

```diff
@@ -179,120 +170,79 @@ Now consider the following refinement to the query:
 
 [source, cypher]
 ----
+PROFILE
 MATCH (p:Person)-[:ACTED_IN]->(m:Movie) WHERE p.name STARTS WITH 'Tom'
 RETURN p.name, count(m) ORDER BY p.name
 ----
 
-//table
+//profile
 
 We are asking for the results in ascending alphabetical order.
 The native index happens to store String properties in ascending alphabetical order, and Cypher knows this.
 In Neo4j 3.4 and earlier, Cypher would plan a `Sort` operation to sort the results, which means building a collection in memory and running a sort algorithm on it.
 For large result sets this can be expensive in terms of both memory and time.
-If you are using the native index, Cypher will recognise that the index already returns data in the correct order, and skip the `Sort` operation.
-
-Indexes storing values of the spatial type `Point`, and non-native indexes, cannot be relied on to return the values in the correct order.
-This means that for Cypher to enable this optimization, the query needs a predicate that limits the type of the property to some type that is guaranteed to be in the right order.
-
-To demonstrate this effect, let's remove the String prefix predicate so that Cypher no longer knows the type of the property, and replace it with an existence predicate.
-Now the database can no longer guarantee the order.
-If we profile the query we will see the `Sort` operation:
-
-[source, cypher]
-----
-PROFILE
-MATCH (p:Person)-[:ACTED_IN]->(m:Movie) USING INDEX p:Person(name) WHERE exists(p.name)
-RETURN p.name, count(m) ORDER BY p.name
-----
-
-//profile
+In Neo4j 3.5 and later Cypher will recognise that the index already returns data in the correct order, and skip the `Sort` operation.
```

Comment:
```suggestion
In Neo4j 3.5 and later, Cypher will recognize that the index already returns data in the correct order, and skip the `Sort` operation.
```

Reply:
Morning il new user please i need yours help to continue